### PR TITLE
chore: update issue templates

### PR DESCRIPTION
**Pull request by raisedadead**
This PR

- Updates the issue creation page

  image

- Updates the labels (which I will adjust once this lands)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/03--issues-with-content-on-articles-and-docs.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-name: 'Bug Report: Issues with /news or guide articles or documentation'
-about: Report issue with a specific article, like broken links, typos, missing parts, etc
+name: 'Report issues with content on /news, guide articles or documentation'
+about: Report issues with content on a specific article, like broken links, typos, missing parts, etc.
 title: ''
-labels: 'status: needs help for triage, type: bug'
+labels: 'type: bug, status: waiting triage'
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/03--issues-with-content-on-articles-and-docs.md:5]: Do we want to update the label name then?

Comment (raisedadead) [reply .github/ISSUE_TEMPLATE/03--issues-with-content-on-articles-and-docs.md:5]: Yes - You are correct. I mentioned in the OP, I intend to adjust the labels after this lands.

File: .github/ISSUE_TEMPLATE/01--issues-with-coding-challenges.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Add a clear and concise description of what you expected to happen.
+
+
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem. you can drag and drop, png, jpg, gif, etc. in this box.
+
+
+
+**Smartphone (please complete the following information):**
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/01--issues-with-coding-challenges.md:38]:
```suggestion
**System (please complete the following information):**
```

There might be a better word than `System` but I'm not sure that `Smartphone` fits here. 🙂

File: .github/ISSUE_TEMPLATE/02--issues-with-software-on-platforms.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+A clear and concise description of what you expected to happen.
+
+
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem. you can drag and drop, png, jpg, gif, etc. in this box.
+
+
+
+**Smartphone (please complete the following information):**
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/02--issues-with-software-on-platforms.md:39]:
```suggestion
**System (please complete the following information):**
```

**Review by naomi-lgbt — COMMENTED** [review]
A quick nitpick and then lgtm

**Review by naomi-lgbt — APPROVED** [review]
Looks good 🙂

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**